test/font/fontutils.js

```diff
@@ -88,3 +88,7 @@ function verifyTtxOutput(output) {
     throw m[1];
   }
 }
+
+exports.decodeFontData = decodeFontData;
```

Comment:
I don't really understand these changes. There is no `exports` variable in this file. Could you explain this a bit?